app/src/lib/stores/app-store.ts

```diff
@@ -504,16 +500,12 @@ export class AppStore extends TypedBaseStore<IAppState> {
       error => this.emitError(error)
     )
 
-    const browserWindow = remote.getCurrentWindow()
-    this.windowState = getWindowState(browserWindow)
```

Comment:
Wouldn't we want to initialize this (even if not synchronously?)

Reply:
Seeing your other comment, I guess you saw that I am initializing this as I was sending these values on app-window load. Tho I didn't know the concern about that possibly loading before the app-store since it tested out well.  But, no matter, working to use the invoke/handle to request the values from here (2nd stab at that.. was running into issue the first time I tried that).